src/js/dp/search.js

```diff
@@ -8,138 +8,173 @@ const fetchHtml = async (url) => {
       Accept: "application/json",
     }),
   });
-  return response && await response.text();
+  return response && (await response.text());
```

Comment:
If `await fetch()` rejects then it'll throw on line 4, so testing the truthiness of the reponse isn't needed in `response &&`. However, `fetch()` won't reject on a server error so you do have to check for `response.ok` before continuing.

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API#differences_from_jquery

Reply:
Agreed but I am removing the `console.error` by adding a guard clause/iffy and cannot test the refactor to an adequate level to ensure the changes wouldn't have a significant impact. These changes have appeared as I ran a formatter